src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/PackageAnalyzersDependencyNode.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using Microsoft.VisualStudio.Imaging;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    public class PackageAnalyzersDependencyNode : DependencyNode
```

Comment:
Yes, i was thinking about that too and hesitated what would be better :).. Ok, i will keep analyzerassemblies at the same level as other assemblies.

---
In reply to: [76305954](https://github.com/dotnet/roslyn-project-system/pull/374#discussion_r76305954) [](ancestors = 76305954)